optimus/src/main/java/co/mobiwise/optimus/Optimus.java

```diff
@@ -171,7 +173,17 @@ public static boolean contains(final String key) {
   public static Editor getEditor() {
     return getOptimusPref().edit();
   }
+  
+  public static void putStringListWithName(String key, List<String> list){
```

Comment:
isimlendirme de bence metot adında putList olarak bırakıp metot parametresinden ilerlemek daha dogru bir yaklasım olur. zaten parametre olarak generic bir yapı ihtiyacı var. orada isimlendirme daha anlamlı olmus oluyor.

Reply:
putStringSet metodunu örnek aldığım için böyle yaptım. Generic bir yapı daha iyi olur tabii.

Reply:
putStringSet adını buna benzer degistireğim 👍